cfgov/v1/atomic_elements/organisms.py

```diff
@@ -17,7 +17,7 @@
 
 from . import atoms, molecules
 from .. import blocks as v1_blocks
-from ..models.snippets import Contact as ContactSnippetClass
+from v1.models.snippets import Contact as ContactSnippetClass, ReusableText
```

Comment:
Nitpick: this import line bugs me with the combined "import as" and straight "import".